moj_auth/__init__.py

```diff
@@ -115,3 +116,7 @@ def logout(request):
 
     if hasattr(request, 'user'):
         request.user = MojAnonymousUser()
+
+
+def urljoin(base, *parts):
+    return '/'.join([s.strip('/') for s in [base] + list(parts)]) + '/'
```

Comment:
Yes.

a) ```urllib.parse.urljoin``` does not accept a variable number of arguments, so for multiple parts one would have to do ```urljoin(urljoin(base, part1), part2)``` which is just kind of a hassle.
b) ```urllib.parse.urljoin``` does not end the generated URL with a forward slash, which is the default in Django, and also in the ```responses``` library that we use for testing.
c) The main extra thing that ```urllib.parse.urljoin``` will do beyond the functionality of this function is that if called with e.g. ```urljoin('http://root/url/', '/path')``` it will return ```http://root/path``` which is not a piece of functionality that I particularly want in this use case.